### PR TITLE
rename CS MI and SA to clusters-service

### PR DESCRIPTION
**Pull request by geoberle**
### What this PR does

this is for consistency reasons with non-ARO-HCP CS deployments

Jira: https://issues.redhat.com/browse/ARO-8949
Link to demo recording:

### CODE REVIEW

File: cluster-service/deploy/openshift-templates/arohcp-service-template.yml [hunk 1]
```diff
@@ -214,6 +214,9 @@ parameters:
 - name: CPU_LIMIT
   description: CPU limit.
   value: "1"
+- name: CS_MI_CLIENT_ID
```
Comment (JameelB) [inline cluster-service/deploy/openshift-templates/arohcp-service-template.yml:217]: Should we prefix this with `AZURE`? For cloud specific parameters, I think it's prefixed with the associated cloud, e.g. `AWS_STS_POLICY_DIRECTORY`.  

Can also be done in a follow up PR if we do decide if we want to add this.

**Review by JameelB — APPROVED** [review]
lgtm, thanks @geoberle. 

Have one minor comment around naming of the managed identity client id parameter but that can be followed up in a separate pr.

**Review by tonytheleg — APPROVED** [review]

File: cluster-service/deploy/openshift-templates/arohcp-service-template.yml (hunk 1 above)
Comment (geoberle) [reply cluster-service/deploy/openshift-templates/arohcp-service-template.yml]: sure. fixed

**Review by bennerv — APPROVED** [review]

**Review by jmelis — APPROVED** [review]
LGTM

**Pull request merged**